Representable/representable
# Delete drives

**Changes:**
- delete button for drives
- pop-up modal to confirm deletion of drive
- change CommunityEntry Drive foreign key's ondelete property so that communities don't get deleted when drives are deleted 
- URL for deleting a drive
- view in views/dashboard.py for deleting a drive

**To test:**
- migrate
- runserver
- create a drive to delete (as org admin) (or just delete one you don't want anymore)
- add a community to the drive
- navigate to the drive page (/dashboard/partners/<org-slug>-<org-pk>/ -> click manage on the drive)
- press the red trash button to delete
- check that both the cancel and close buttons dismiss the modal and don't delete the drive
- click delete permanently
- check that the drive no longer exists in the db
- check that the community you added to that drive still exists, but has drive=NULL now

Closes #160
